usr/bin/torbrowser

```diff
@@ -197,9 +212,10 @@ tb_templatevm_check() {
    fi
 
    if [ "$have_other_args" = "false" ]; then
-      local MSG="<p>Do not run Tor Browser in TemplateVM.<br></br>
+      tb_qubes_wiki=Running_"$tb_wiki"_in_Qubes_TemplateVM
```

Comment:
Minor. Looks strange.

Should be:

    tb_qubes_wiki="Running_${tb_wiki}_in_Qubes_TemplateVM"